book/src/tutorial/hardware.md

```diff
@@ -0,0 +1,64 @@
+# Running on Hardware
+
+To run on real hardware you'll need a specially-prepared USB drive.
+
+## Preparation
+
+The general steps to prepare the drive are:
+
+1. Partition the drive using [GPT].
```

Comment:
I don't know if this might be overkill?

For me, it is sufficient to use a simple "convert as FAT" in the Windows Explorer, copy the file to `/efi/boot/bootx64.efi` and it works.

Reply:
Good point, on many UEFI implementations this is not strictly required. Section 13.3.3 says: "UEFI implementa-
tions may allow the use of conforming FAT partitions which do not use the ESP GUID". I've updated the text to note that in many cases its sufficient just to have a FAT partition with the right directory structure.

Reply:
nice, thanks!